olumidae/WayFarer
# #167082674

**What does this PR do?**
Refactor user, bus and trip controller 

**Description of the task to be completed?**
Reduce code smells and duplication 

**How should this be manually tested?**
Clone the repo cd into it and run,
Check the user, bus and trip controller, each method should not be more than 25 lines of code

**Any background context you want to provide?**
Not applicable

**What are the relevant pivotal tracker stories?**
#167082674
